receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,533 @@
+    type: string
+    enum:
+      - transmitted
+      - received
+  vm_count_power_state:
+    value: power_state
+    description: Whether the virtual machines are powered on or off
+    type: string
+    enum:
+      - "on"
+      - "off"
+  vsan_latency_type:
+    value: type
+    description: The type of vSAN latency.
+    type: string
+    enum:
+      - read
+      - write
+  vsan_operation_type:
+    value: type
+    description: The type of vSAN operation.
+    type: string
+    enum:
+      - read
+      - write
+      - unmap
+  vsan_throughput_direction:
+    value: direction
+    description: The type of vSAN throughput.
+    type: string
+    enum:
+      - read
+      - write
+
+metrics:
+  vcenter.cluster.cpu.available:
+    enabled: true
+    description: The amount of CPU available to the cluster
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.cpu.effective:
+    enabled: true
+    description: The effective CPU available to the cluster. This value excludes memory from hosts in maintenance mode or are unresponsive.
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.cpu.used:
+    enabled: true
+    description: The amount of CPU used by the cluster
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.available:
+    enabled: true
+    description: The available memory of the cluster.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.effective:
+    enabled: true
+    description: The available memory of the cluster.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.used:
+    enabled: true
+    description: The memory that is currently used by the cluster
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.vsan.latency.avg:
+    enabled: true
+    description: Average latency of IOs generated by all vSAN clients in the cluster.
+    unit: µs
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [vsan_latency_type]
+  vcenter.cluster.vsan.operations:
+    enabled: true
+    description: IOPS consumed by all vSAN clients in the cluster
+    unit: "{operations/sec}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [vsan_operation_type]
+  vcenter.cluster.vsan.congestions:
+    enabled: true
+    description: Congestions consumed by all vSAN clients in the cluster, such as virtual machines, stats objects, etc.
```

Comment:
Is "congestions consumed" a vCenter term?

Reply:
Yes, specifically a vSAN term.

> Whenever there is a bottleneck in a lower layer of vSAN (closer to the physical storage devices), vSAN uses this flow control (aka congestion) mechanism to relieve the bottleneck in the lower layer and instead reduce the rate of incoming I/O at the vSAN ingress